docs/lib/datastore/fragments/native_common/setup-auth-rules.md

```diff
@@ -0,0 +1,11 @@
+The `@auth` directive can be used to describe the authorization configuration parameters for each of the models.
```

Comment:
I think this is too technical and obscure for the first line. Maybe something simple like:

"Amplify gives you the ability to limit create, read, update, and delete operations on your data to specific individuals or groups."

Reply:
how bout this:

```suggestion
Amplify gives you the ability to limit which individuals or groups should have access to create, read, update, or delete data on your various schema types by specifying an `@auth` directive.
```

this page is a little bare, i would also suggest showing some commonly used ones like

- Everything private: `@auth(rules: [{ allow: owner }])`
- Public read, owner based CRUD: `@auth(rules: [{ allow: owner, queries: null }])`

etc

Reply:
This is so much better @sw-yx! Nice improvement.

Reply:
@sw-yx - like this idea, I was going down the path of heavily documenting each and every feature but ultimately aborted from it.... but your "light-weight" examples are kind of what we are looking for on this page.

i'll take a stab at doing this, and repost updates.